core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3804,20 +3865,23 @@ public String toString() {
                   + "%s using the port defined in %s",
               Name.MASTER_RPC_ADDRESSES, Name.JOB_MASTER_RPC_PORT,
               Name.JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES, Name.JOB_MASTER_RPC_PORT))
-          .setScope(Scope.CLIENT)
+          .setScope(Scope.MASTER)
```

Comment:
shouldn't this be `Scope.ALL`? Clients need that information to connect to the job master? Mainly for CLI use cases

Reply:
Changed, thanks